tests/serializers/tests.py

```diff
@@ -340,6 +341,13 @@ def test_serialize_proxy_model(self):
         self.assertEqual(base_data, proxy_data.replace('proxy', ''))
         self.assertEqual(base_data, proxy_proxy_data.replace('proxy', ''))
 
+    def test_serialize_inheritance(self):
+        child_object = Child.objects.create(parent_data=2, child_data=3)
+        child_object.parent_m2m.add(Parent.objects.create(parent_data=4))
```

Comment:
`parent_data` and `child_data` are `CharField`\`s, so maybe `'a'`, `'b'`, ... etc.